src/components/ListContainer/index.tsx

```diff
@@ -11,17 +11,17 @@ interface Props {
 }
 
 export function ListContainer({ taskArray, isCompleted, toggle }: Props) {
-  const statusAndHasTasks = () => {
-    if (isCompleted) {
-      return <h4>Completed Tasks</h4>
-    } else if (!isCompleted) {
-      return <h4>Active Tasks</h4>
-    }
-    return null
-  }
   return (
     <div className="list-container">
```

Comment:
Food for thought:

Consider matching your component name with the root classname:
```suggestion
    <div className="ListContainer">
```

This allows for the relationship and structure to be cleanly defined like this:
```css
.ListContainer {}
.ListContainer-heading {}
.ListContainer-tasks {}
```
Notice that `CamelCased` words denote elements and `-` denote that they are a sub-structure.